integrationTesting/tests/organize/surveys/submitting-survey.spec.ts

```diff
@@ -0,0 +1,71 @@
+import { expect } from '@playwright/test';
+
+import KPD from '../../../mockData/orgs/KPD';
+import KPDMembershipSurvey from '../../../mockData/orgs/KPD/surveys/MembershipSurvey';
+import RosaLuxemburg from '../../../mockData/orgs/KPD/people/RosaLuxemburg';
+import RosaLuxemburgUser from '../../../mockData/users/RosaLuxemburgUser';
+import test from '../../../fixtures/next';
+
+test.describe('User submitting a survey', () => {
```

Comment:
I think more tests should be added, including:

* Signing survey with name and email
* Signing survey anonymously
* Handling an upstream (API) error, which can be simulated by an API mock that returns an error (or just not setting an API mock at all for `…/submissions`

Reply:
Fixed in 7c7976e896c916a8fc7e0645ccabd69be3484388! Look for `submits user signature`, `submits anonymous signature` and `preserves inputs on error`!